py-rse/teams.Rmd

```diff
@@ -33,131 +39,158 @@ The project directory should now include:
-we need to have a mechanism through which people can contribute.
-Moreover, we need to effectively communicate that project participants enthusiastically desire contributions,
-and that newcomers to the project are valued regardless of their background or identity.
-In other words, a supportive environment for contribution depends on recognizing
-the privilege and consequent oppression that exist in many software development communities,
-and actively taking steps to ensure everyone feels welcome to contribute.
-
-[Privilege][privilege] is an unearned advantage given to some people but not all,
-while [oppression][oppression] is systemic, pervasive inequality that benefits the privileged
-and harms those without privilege.
-A straight, white, physically able, economically secure male
-is less likely to be interrupted when speaking,
-more likely to be called on in class,
-and more likely to get a job interview based on an identical CV
-than someone who is perceived as being outside these categories.
-People who are privileged are often not aware of it,
-as they've lived in a system that provides unearned advantages their entire lives.
-A [target][target] is someone who suffers from oppression.
-Targets are often called "members of a marginalized group",
-but that phrasing is deliberately passive.
-Targets don't choose to be marginalized:
-those with privilege marginalize them.
-An [ally][ally] is a member of a privileged group
-who is working to understand their own privilege and end oppression.
+Most research software projects begin as the work of one person,
+who may continue to do the bulk of the coding and data analysis throughout its existence @Maju2019.
+As projects become larger,
+though,
+they eventually need more contributors to sustain them.
+Involving more people also increases the functionality and robustness of the code,
+since newcomers bring their own expertise or see old problems in new ways.
+In order to leverage a group's expertise,
+though,
+a project must do more than *allow* people to contribute:
+its leaders must communicate that the project *wants* contributions,
+and that newcomers are welcome and valued @Shol2019.
+
+But saying "the door is open" is not enough,
+since many potential contributors have painful personal experience of being less welcome than others.
+In order to create a truly welcoming environment for everyone,
+the project must explicitly acknowledge that some people are treated unfairly
+and actively take steps to remedy this.
+Doing this increases diversity within the team,
+which makes it more productive @Zhan2020.
+More importantly,
+it is the right thing to do.
+
+> **Terminology**
+>
+> [Privilege][privilege] is an unearned advantage given to some people but not all,
+> while [oppression][oppression] is systemic inequality that benefits the privileged
+> and harms those without privilege @Auro2019.
+> In Europe, Australia, and the Americas,
+> a straight, white, affluent, physically able male
```

Comment:
I was going to suggest "Australasia", but [then read](https://en.wikipedia.org/wiki/Australasia) that that means different things to Australians and New Zealanders.  I think we could add NZ to the list...

Reply:
amended